src/Utils.hs

```diff
@@ -176,11 +176,13 @@ parens = between (symbol "(") (symbol ")")
 
 -- | 'integer' parses an integer.
 integer :: Parser Integer
-integer = lexeme L.decimal
+unsignedInteger       = lexeme L.decimal
```

Comment:
We try to have type signatures for all top-level functions in the codebase, so it would be good to add one for `unsignedInteger` and `unsignedFloat`

Reply:
I'm not sure I understand what do you mean here

Reply:
Looking at the diff, it looks like you don't have type signatures for `unsignedInteger` or `unsignedFloat`. Could you add them?